platform.md

```diff
@@ -157,11 +597,6 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
-## Exported Image
```

Comment:
was this moved to somewhere else that I missed?

Reply:
This section doesn't exist in the current spec on the `main` branch. I get the impression it was added mainly to provide a place to describe the project metadata label. I thought about expanding on this section rather than removing it but decided against it (after some equivocating) on the theory that the content would mostly be a repetition of the `exporter` section.

I am happy to add something like this back (I would probably call it `App Image` to match the terminology on which I have been attempting to standardize)